frontend/src/utils/rulesRegex.ts

```diff
@@ -24,5 +24,11 @@ export const rules: IFormRules = reactive({
     'Value is not a valid license plate value',
   vinNumber: (value) => /^[\dA-Za-z]{17}$/.test(value) ? 
     true : 
-    'Value is not a valid VIN number',
+    'Value is not a valid VIN number'
+});
+
+export const rulesFiles: IFormFilesRules = reactive({
+  photoSizeMax: (value) => !value || !value.length || value[0]?.size < 10000000 ? 
+    true :
+    'A photo should be less than 10 MB'

```

Comment:
10000000 Bytes = 9.5367431640625 MB (in binary)

Reply:
Done